uaainvite/templates/email/body.html

```diff
@@ -7,71 +7,35 @@
     <body>
         <p>Hello!</p>
         <p>
-            admin has invited you to join the
-            closed pilot of
-            <a href="https://cloud.gov">{{branding.company_name}}</a>
+            You've been invited to join 
+            <a href="https://cloud.gov">{{branding.company_name}}</a>.
```

Comment:
Actually - I take that back @brittag.  We don't have a separate public site for e/w and govcloud.  This is fine.